docs/test_configuration.rst

```diff
@@ -2,10 +2,8 @@
  Test Configuration
 ====================
 
-Configuration of *USM QE integration tests* project piggibacks/reuses pytest
-ini-options parser, so that all USM QE configuration options could be specified
-in ``pytest.ini`` file. To prevent a conflict with pytest ini-option names, we
-use ``usm_`` prefix in a name of every option.
+Configuration of *USM QE integration tests* project uses ``YAML configuration
+files``, ``ansible hosts files`` and `pytest.ini`_ file.
```

Comment:
I would drop the quoting here, as it looks as if there were file names ...

eg.:
```suggestion
files``, `ansible hosts files` and `pytest.ini`_ file.
```